Amplify/Categories/API/AuthProvider/APIAuthProviderFactory.swift

```diff
@@ -26,7 +26,11 @@ open class APIAuthProviderFactory {
 
 public protocol AmplifyAuthTokenProvider {
     typealias AuthToken = String
+
+    @available(*, deprecated, message: "Use getUserPoolAccessToken() instead")
```

Comment:
You can use this syntax instead which will let Xcode auto-correct the code.

`@available(*, deprecated, renamed: "getUserPoolAccessToken")`

Reply:
done